Acore_SendAndBind.lua

```diff
@@ -48,8 +48,10 @@ end
 local function SendAndBind(event, player, command, chatHandler)
     local itemGUID
     local item_id
+    local item_id_name
```

Comment:
let's use `item_name` and `target_name` for consistency

Reply:
Like this? (submitted the changes)